test/triplex/triplex_test.exs

```diff
@@ -37,13 +37,17 @@ defmodule TriplexTest do
   test "create/2 must return a error if the tenant already exists" do
     assert {:ok, _} = Triplex.create("lala", PGTestRepo)
 
-    assert {:error, "ERROR 42P06 (duplicate_schema) schema \"lala\" already exists"} =
```

Comment:
This failed for me because the message returned was `"ERROR 42P06 (duplicate_schema) schema \"lala\" already exists\n\n    query: create schema \"lala\";"` - I'm skeptical that is a result of using elixir 1.17 and suspect it is caused by either my version of mysql being different or a configuration setting in my mysql. In my opinion the "meat" of the assertion is preserved in that the error code is still checked.